src/makeCalculator.js

```diff
@@ -3,8 +3,42 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
+function makeCalculator(operation, value) {
   // write code here
+
+  const calculator = {
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
+    },
+
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+
+    result: 0,
+
+    add: function (number) {
+      this.result += number;
+    },
+
+    subtract: function (number) {
+      this.result -= number;
+    },
+
+    divide: function (number) {
+      this.result /= number;
+    },
+
+    multiply: function (number) {
```

Comment:
[CODE KNOWLEDGE] - For the `multiply` method, use the method shorthand. Replace `multiply: function (number) {` with `multiply(number) {`.